api/alembic/versions/fbca68ccc9da_classification_thresholds.py

```diff
@@ -0,0 +1,47 @@
+"""classification thresholds
+
+Revision ID: fbca68ccc9da
+Revises: 00df3c7b5cba
+Create Date: 2022-09-08 13:09:24.894403
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm.session import Session
+
+
+# revision identifiers, used by Alembic.
+revision = 'fbca68ccc9da'
+down_revision = '00df3c7b5cba'
+branch_labels = None
+depends_on = None
+
+classification_threshold_table = sa.Table(
+    'advisory_hfi_classification_threshold',
+    sa.MetaData(),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('description', sa.String(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+)
+
+
+def upgrade():
```

Comment:
This migration takes awhile to grab everything from arcserver, will this migration timeout when we deploy?

Reply:
It's never happened to me - it's certainly possible - you'd have to re-run the migration if it fails.